src/annotator/anchoring/pdf.ts

```diff
@@ -644,6 +644,9 @@ async function anchorShape(
   const pageWidth = pageRight - pageLeft;
   const pageHeight = pageTop - pageBottom;
 
+  const mapX = (x: number) => (x - pageLeft) / pageWidth;
+  const mapY = (y: number) => (pageTop - y) / pageHeight;
```

Comment:
The main change here is to use `(pageTop - y) / pageHeight` instead of `(pageHeight - y) / pageHeight` in Y conversions.